Mathlib/RingTheory/Algebraic.lean

```diff
@@ -712,15 +711,15 @@ theorem Algebra.IsAlgebraic.exists_smul_eq_mul [IsDomain S] [Algebra.IsAlgebraic
 
 end
 
-variable {R S : Type*} [CommRing R] [IsDomain R] [CommRing S]
+variable {R S : Type*} [CommRing R] [CommRing S]
 
 theorem exists_integral_multiple [Algebra R S] {z : S} (hz : IsAlgebraic R z)
     (inj : ∀ x, algebraMap R S x = 0 → x = 0) :
-    ∃ᵉ (x : integralClosure R S) (y ≠ (0 : R)), z * algebraMap R S y = x := by
+    ∃ᵉ (x : integralClosure R S) (y ≠ (0 : R)), algebraMap R S y * z = x := by
   rcases hz with ⟨p, p_ne_zero, px⟩
   set a := p.leadingCoeff
   have a_ne_zero : a ≠ 0 := mt Polynomial.leadingCoeff_eq_zero.mp p_ne_zero
-  have x_integral : IsIntegral R (z * algebraMap R S a) :=
+  have x_integral : IsIntegral R (algebraMap R S a * z) :=
```

Comment:
Splitting of the file [RingTheory/Algebraic.lean](https://github.com/leanprover-community/mathlib4/pull/6183/files#diff-1fed4ce28689dbfee7313dcc400e1b532d2bc3e97b932d74791f8506e07f9083) #19370 was just merged into master, so you need to merge master again ...

Reply:
And which introduce big changes.

Reply:
What do you mean by big changes?

Reply:
Oh, I mean that file is split into several files, so the newly added results in `Algebraic` need to find suitable places. But since now the merge conflict is resolved, so I think this is done.

Reply:
Yeah, results in my algebraicClosure branch may need to go into multiple files. But in this PR only one lemma is changed in the Algebraic file.